app/workers/activitypub/update_distribution_worker.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 class ActivityPub::UpdateDistributionWorker < ActivityPub::RawDistributionWorker
-  sidekiq_options queue: 'push', lock: :until_executed
```

Comment:
Why?

Reply:
This sidekiq_options was added at https://github.com/mastodon/mastodon/pull/19380/commits/00b7b9573acd746a366dd01f0abad88417afcbbc.

With this fix, this worker is no longer needed as it is no longer run in perform_in.
But you might not have to remove.

Reply:
Undo it.